SECURITY.md

```
@@ -94,22 +94,29 @@ Apply the same pattern for any new GitHub Actions `pip install` steps so Scoreca
 
 ```bash
-slsa-verifier verify-artifact eval-hub-vX.Y.Z.tar.gz \
-  --provenance-path multiple.intoto.jsonl \
-  --source-uri github.com/eval-hub/eval-hub \
-  --source-tag vX.Y.Z
+gh attestation verify eval-hub-vX.Y.Z.tar.gz \
+  --repo eval-hub/eval-hub
 ```
 
-`SHA256SUMS` is an integrity aid for the archive; it is not a Scorecard-recognized signature. The Scorecard-recognized artifact is the `*.intoto.jsonl` provenance file.
+3. Or verify offline with the Sigstore bundle from the release:
```

markdownlint-cli2 (0.23.2)

[warning] Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/1/1

(MD029, ol-prefix)